tests/test_codecs.py

```diff
@@ -111,7 +113,7 @@ def test_context_to_readable_headers(self):
             assert carrier == {'trace-id': '100:7f:0:1'}
 
             ctx._baggage = {
-                'fry': 'Leela',
+                'fry': u'Leela',
```

Comment:
Could you also update the string to have a unicode character which doesn't have an ascii representation?

Reply:
there is a new test that does that

Reply:
I don't think that the new test, `test_non_ascii_baggage` verifies that the carrier carries the correctly encoded baggage string.